src/frontend/web_application/src/components/ContactDetails/components/FormSelector/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { SelectFieldGroup } from '../../../form';
+
+import './style.scss';
+
+class FormSelector extends Component {
+  static propTypes = {
+    __: PropTypes.func.isRequired,
+    formsOptions: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  };
+
+  state = {
+    selectedForm: null,
+  }
+
+  handleSelectChange = (ev) => {
+    const { value } = ev.target;
+    this.setState(prevState => ({
+      ...prevState,
+      selectedForm: value,
+    }));
+  }
+
+  renderSelector = () => {
+    const { __, formsOptions } = this.props;
+    const typeOptions = formsOptions.map(option => ({
+      value: option.name,
+      label: option.name,
+    }));
+
+    return (
+      <SelectFieldGroup
+        name="selectedForm"
+        className="m-form-selector__select"
+        value=""
```

Comment:
the state would be a bit difficult to work with. I'll see what to do when implementing api